src/status.rs

```diff
@@ -523,15 +521,6 @@ impl PartialEq for StatusCode {
     }
 }
 
-impl Eq for StatusCode {}
-
-impl Clone for StatusCode {
-    #[inline]
-    fn clone(&self) -> StatusCode {
-        *self
-    }
-}
-
```

Comment:
These 3 manual impls were because the auto derives noticeably affected compile times. Is that no longer the case?